packages/teleport/src/Discover/SelectResource/ServerResource.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import { Link, Text } from 'design';
+
+import { ActionButtons, TextBox } from 'teleport/Discover/Shared';
+
+import { PermissionsErrorMessage } from './PermissionsErrorMessage';
+
+export function ServerResource(props: ServerResourceProps) {
+  let content = <TeleportVersions />;
+  if (props.disabled) {
+    content = (
+      <PermissionsErrorMessage
+        action="add new Servers"
+        productName="Server Access"
+      />
+    );
+  }
+
+  return (
+    <>
+      {content}
+
+      <ActionButtons
+        onProceed={() => props.onProceed()}
+        disableProceed={props.disabled}
+      />
+    </>
+  );
+}
+
+function TeleportVersions() {
+  return (
+    <TextBox data-testid="server-teleport-versions">
+      <Text typography="h5">
+        Teleport officially supports the following operating systems:
+      </Text>
+      <ul style={{ paddingLeft: 28 }}>
+        <li>Ubuntu 14.04+</li>
+        <li>Debian 8+</li>
+        <li>RHEL/CentOS 7+</li>
+        <li>Amazon Linux 2</li>
+        <li>macOS (Intel)</li>
```

Comment:
As the M1 support is coming in the near future could you create an issue to track this update plz.

Reply:
Done - #1167